src/lib/coda_base/data_hash.ml

```diff
@@ -80,9 +80,18 @@ struct
   module Stable = struct
     module V1 = struct
       module T = struct
-        (* TODO : version Pedersen.Digest *)
+        (* TODO : will this type be versioned? *)
         type t = Pedersen.Digest.t
-        [@@deriving bin_io, sexp, eq, compare, hash, yojson, version]
+        [@@deriving bin_io, sexp, eq, compare, hash, version]
+
+        type yojson = {value: Pedersen.Digest.t; logproc_interp: string}
```

Comment:
Can you make `logproc_interp` a constant. I was confused for a second and thought that `logproc_interp` can be different values

Reply:
How do you suggest making it constant? The only place values of this type are created, it is a literal. I'm only defining the type so I don't have to write out `to_yojson` by hand.

Reply:
I was thinking about doing something like 
```ocaml
module Constants = struct
    let logproc_interp = "data_hash"
end

type yojson = {value: Pedersen.Digest.t; logproc_interp: string}

let to_yojson value = yojson_to_yojson {value; logproc_interp=Constants.logproc_interp}
```